app/presenters/tree_builder.rb

```diff
@@ -310,6 +310,7 @@ def count_only_or_many_objects(count_only, *collections)
 
   def count_only_or_objects(count_only, objects, sort_by = nil)
     if count_only
+      objects = objects.except(:order) if objects.respond_to?(:order)
```

Comment:
We love functional programming and reassigning a variable is a bad thing in my world, could you please rather write:
```suggestion
      objects.respond_to?(:order) ? objects.except(:order).size : objects.size
```